go/stellar/stellarsvc/frontend.go

```diff
@@ -1241,7 +1241,7 @@ func (s *Server) FindPaymentPathLocal(ctx context.Context, arg stellar1.FindPaym
 	res.ExchangeRate = fmt.Sprintf("%s = %s", exchangeRateLeft, exchangeRateRight)
 
 	if len(path.SourceInsufficientBalance) > 0 {
-		availableToSpend, err := stellar.FormatAmountDescriptionAssetEx(mctx, path.SourceInsufficientBalance, path.SourceAsset)
+		availableToSpend, err := stellar.FormatAmountDescriptionAssetEx2(mctx, path.SourceInsufficientBalance, path.SourceAsset)
```

Comment:
@patrickxb I added a FormatAmountDescriptionAssetEx2 for "domain or accountID" formatting

Reply:
Sounds good